packages/webapp/webapp_server.js

```diff
@@ -484,23 +484,33 @@ function getStaticFileInfo(originalPath, browser) {
     return null;
   }
 
-  if (hasOwn.call(staticFilesByArch, arch)) {
+  // Get a list of all available static file architectures, with arch
+  // first in the list if it exists.
+  const staticArchList = Object.keys(staticFilesByArch);
+  const archIndex = staticArchList.indexOf(arch);
+  if (archIndex > 0) {
+    staticArchList.unshift(staticArchList.splice(archIndex, 1)[0]);
+  }
+
+  let info = null;
+
+  staticArchList.some(arch => {
     const staticFiles = staticFilesByArch[arch];
 
     // If staticFiles contains originalPath with the arch inferred above,
     // use that information.
     if (hasOwn.call(staticFiles, originalPath)) {
-      return staticFiles[originalPath];
+      return info = staticFiles[originalPath];
```

Comment:
can you simplify this by using `.find` instead of `.some`?

Reply:
It would be a little weird because we started with an array of `arch`s, but we want to end up with either `staticFiles[originalPath]` or `staticFiles[path]`, neither of which is a an `arch` from the original array.